R/TaskClassif_penguins.R

```diff
@@ -35,7 +35,7 @@ load_task_penguins = function() {
   penguins = as.data.table(palmerpenguins::penguins)
   setnames(penguins,
     old = c("bill_length_mm", "bill_depth_mm", "flipper_length_mm", "body_mass_g"),
-    new = c("bill_length",    "bill_depth",    "flipper_length",    "body_mass")
+    new = c("bill_length", "bill_depth", "flipper_length", "body_mass")
```

Comment:
@lorenzwalthert DO you have an idea why this alignment is not honoured?

Reply:
Because spacing is only honoured one level deep. This is two level deep (function call in function call). Sorry, styler can't do this. Use `strict = FALSE` if you want to preserve one more more space here (but it comes with other implications as shown in the dedicated vignette).